Packages/com.unity.inputsystem/Tests/InputSystem/Plugins/SwitchTests.cs

```diff
@@ -14,25 +14,16 @@ public void Devices_SupportsSwitchNpad()
         var device = InputSystem.AddDevice(
                 new InputDeviceDescription
         {
-            interfaceName = "NPad",
+            interfaceName = "Switch",
             manufacturer = "Nintendo",
             product = "Wireless Controller",
         });
 
         Assert.That(device, Is.TypeOf<NPad>());
         var controller = (NPad)device;
 
-        InputSystem.QueueStateEvent(controller,
-            new NPadInputState()
-            .WithLeftStick(new Vector2(0.123f, 0.456f))
-            .WithRightStick(new Vector2(0.789f, 0.987f)));
         InputSystem.Update();
 
-        Assert.That(controller.leftStick.x.ReadValue(), Is.EqualTo(0.123).Within(0.00001));
-        Assert.That(controller.leftStick.y.ReadValue(), Is.EqualTo(0.456).Within(0.00001));
-        Assert.That(controller.rightStick.x.ReadValue(), Is.EqualTo(0.789).Within(0.00001));
-        Assert.That(controller.rightStick.y.ReadValue(), Is.EqualTo(0.987).Within(0.00001));
```

Comment:
Please add this back or add something equivalent. This tests whether the sticks come through on the gamepad.